src/Autodesk.Forge.Core/ServiceCollectionExtensions.cs

```diff
@@ -37,6 +37,7 @@ public static IHttpClientBuilder AddForgeService(this IServiceCollection service
         {
             services.AddOptions();
             services.Configure<ForgeConfiguration>(configuration.GetSection("Forge"));
```

Comment:
@szilvaa is there some way to get a warning on this too? e.g. if the user still defines `Forge` in appsettings.json

Reply:
We would need to add our own `.targets` file to the nuget package that would check the user appsettings.json to see if they still use this old stuff. So, it is possible, but it isn't trivial.